services/blockchain-app-registry/shared/utils/paramUtils.js

```diff
@@ -17,9 +17,11 @@ const {
 	Exceptions: { ValidationException },
 } = require('lisk-service-framework');
 
-const EMPTY_STRING = '';
-
 const normalizeRangeParam = (params, property) => {
+	if (!params || typeof params !== 'object') return params;
```

Comment:
Please use `!isObject(params)` from Lisk Service Framework (Utils) instead of `typeof params !== 'object'`.
`typeof []` will also result in `'object'`.

```suggestion
	if (!params || typeof params !== 'object') return params;
```

Reply:
Applied.